lib/bookingsync/api/client/attachments.rb

```diff
@@ -0,0 +1,54 @@
+module BookingSync::API
+  class Client
+    module Attachments
+      # List attachments
+      #
+      # Returns all attachments supported in BookingSync.
```

Comment:
that sounds strange, isn't sth like `Returns attachments for the account, user is authenticated with.` or so?

Reply:
Yeah, little copy/paste there :) I'll change it.